src/Microsoft.Extensions.Localization/ResourceManagerStringLocalizer.cs

```diff
@@ -207,7 +210,11 @@ private IEnumerable<string> GetResourceNamesFromCultureHierarchy(CultureInfo sta
                         resourceNames.Add(resourceName);
                     }
                 }
-                catch (MissingManifestResourceException) { }
+                catch (MissingManifestResourceException)
```

Comment:
In `GetResourceNamesForCulture` you're manually throwing and then catching here. Is there another call site that's throwing `MissingManifestResourceException` or could you just return false and flow that response up? Try -> catching like this can be very expensive and inneficient given what you do below :smile: